authentik/stages/password/stage.py

```diff
@@ -161,7 +161,7 @@ def challenge_invalid(self, response: PasswordChallengeResponse) -> HttpResponse
         ):
             self.logger.debug("User has exceeded maximum tries")
             del self.request.session[SESSION_KEY_INVALID_TRIES]
-            return self.executor.stage_invalid()
+            return self.executor.stage_invalid(_("Invalid password."))
```

Comment:
Should it 
```suggestion
            return self.executor.stage_invalid(_("Invalid password"))
```
nitpick to be consistent with above